gh_actions_exporter/cost.py

```diff
@@ -0,0 +1,18 @@
+from githubkit.webhooks import WorkflowJob
+
+from gh_actions_exporter.config import Settings
+
+
+class Cost(object):
+    def __init__(self, settings: Settings):
+        self.settings: Settings = settings
+
+    def get_job_cost(self, job_request: WorkflowJob, flavor_label: str) -> float:
+        cost_per_min: float = self.settings.job_costs.get(
+            flavor_label, self.settings.default_cost
+        )
+        duration: int = int(job_request.completed_at.timestamp()) - int(
+            job_request.started_at.timestamp()
+        )
+
+        return duration / 60 * cost_per_min
```

Comment:
I see no need of adding `request` into this var name, could be `workflow_job` or just `job`
```suggestion
    def get_job_cost(self, workflow_job: WorkflowJob, flavor_label: str) -> float:
        cost_per_min: float = self.settings.job_costs.get(
            flavor_label, self.settings.default_cost
        )
        duration: int = int(workflow_job.completed_at.timestamp()) - int(
            job_request.started_at.timestamp()
        )

        return duration / 60 * cost_per_min
```